examples/fargate-serverless/versions.tf

```diff
@@ -1,10 +1,10 @@
 terraform {
-  required_version = ">= 1.0.0"
+  required_version = ">= 1.0"
 
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 4.27"
+      version = ">= 4.47"
```

Comment:
Using this version since this is the greatest common denominator (coming from EKS module due to addon custom config)